pkg/action/list.go

```diff
@@ -87,12 +87,14 @@ const ListAll = ListDeployed | ListUninstalled | ListUninstalling | ListPendingI
 type Sorter uint
 
 const (
-	// ByDate sorts by date
-	ByDate Sorter = iota
 	// ByNameAsc sorts by ascending lexicographic order
-	ByNameAsc
+	ByNameAsc Sorter = iota
```

Comment:
is this still in use? If not, is it safe to remove this? I think the `default` case below captures this enum so there's no need to use this if I'm mistaken.

Reply:
Nice catch! It wasn't directly used, but only in test cases to enforce default sort order. I fixed the test cases to remove its usage and adjusted the enum fields.